### PR TITLE
now we can use <display> property

### PR DESCRIPTION
**Pull request by PavelDikiy**
It all started over 14 IOS

### CODE REVIEW

**Comment by thodubois** [comment]
Wow, thanks for your PR. I'll take a look asap.

**Comment by PavelDikiy** [comment]
and need update dependencies for picker 
https://www.npmjs.com/package/@react-native-community/picker - deprecated
https://github.com/react-native-picker/picker - new

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 node_modules/
 npm-debug.log
 yarn-error.log
+.idea
+yarn.lock
```
Comment (thodubois) [inline .gitignore:5]: I don't think it's necessary to remove yarn.lock

**Review by thodubois — CHANGES_REQUESTED** [review]
Just remove the yarn.lock from gitignore or maybe I am missing something about it.  And I will merge it

**Review by thodubois — APPROVED** [review]

**Comment by PavelDikiy** [comment]
i removed the yarn.lock from gitignore

**Comment by PavelDikiy** [comment]
we forgot to update the version (0.2.8 => 0.2.9)

**Comment by thodubois** [comment]
Hi, I pushed a new version, typescript compatible with your change. Some props might have change. Don't hesitate to tell if something is missing.

**Pull request merged**